hrjobcontract/js/src/job-contract/controllers/modal/modal-contract-new.controller.js

```diff
@@ -375,18 +372,13 @@ define([
     /**
      * Reload updated changes for annual deduction and benefit options
      *
-     * @param optionType
-     * @param optionName
-     * @returns {*}
+     * @param {String} optionType
+     * @param {String} optionName
```

Comment:
Wait, why do we remove the `return` here? Please leave it how it was, but in the documentation it should say what it returns. If it is a `{Promise}`, then:

```* @return {Promise} resolved with ....`

Same for other places.